scripts/build.sh

```diff
@@ -13,6 +13,7 @@ function initialize {
 
 function build_index {
   npm run build-index
+  cp index.json pages/
```

Comment:
The `npm run build-index` command was changed to only generate `index.json` in the current directory. The build script duplicates it where needed. This line will be removed once #2865 is fixed.